variables.tf

```diff
@@ -1,15 +1,26 @@
 variable "cluster_name" {
-  type = string
+  description = "Name of the EKS cluster. Must be unique in the AWS account"
+  type        = string
 }
 
 variable "base_domain" {
-  description = "The base domain used for Ingresses."
+  description = <<-EOT
+    The base domain for the cluster.
+
+    This module needs a Route53 zone matching this variable with permission to create DNS records. It will create a wildcard CNAME record `*.apps.<base_domain>` that points to an Elastic Load Balancer routing ingress traffic to all cluster nodes. Such urls will be used by default by other DevOps Stack modules for the applications they deploy (e.g. Argo CD, Prometheus, etc).
```

Comment:
```suggestion
    This module needs a Route 53 zone matching this variable with permission to create DNS records. It will create a wildcard CNAME record `*.apps.<base_domain>` that points to an Elastic Load Balancer routing ingress traffic to all cluster nodes. Such urls will be used by default by other DevOps Stack modules for the applications they deploy (e.g. Argo CD, Prometheus, etc).
```